### PR TITLE
Add 'arrayValue' template that returns the type of values stored in the array

### PR DESCRIPTION
**Pull request by bogdan-szabo-sociomantic**

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Looks like a duplicate of https://github.com/sociomantic-tsunami/ocean/blob/v4.x.x/src/ocean/meta/types/Arrays.d#L37?

File: src/ocean/meta/traits/Arrays.d [hunk 1]
```diff
@@ -71,3 +71,23 @@ unittest
...
+
+    Params:
+        T = any array type
+
+    Returns:
+        The type of values stored in the array
+
+*******************************************************************************/
+
+template arrayValue ( T : U[], U )
```
Comment (ben-palmer-sociomantic) [inline src/ocean/meta/traits/Arrays.d:85]: Wouldn't this be `arrayType`?

File: relnotes/arrayValue.feature.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+### arrayValue
```
Comment (ben-palmer-sociomantic) [inline relnotes/arrayValue.feature.md:1]: This should be a catchy title/description of the addition.

**Review by ben-palmer-sociomantic — COMMENTED** [review]
Made a couple of comments, I have no idea if we already have anything like this in ocean though.

**Comment by bogdan-szabo-sociomantic** [comment]
@gavin-norman-sociomantic yes. I will close this pr

**Pull request closed without merging**